build.gradle.kts

```diff
@@ -56,8 +56,10 @@ dependencies {
     // Below dependencies are solely present so code examples in the test resources dir compile
     testImplementation("javax.validation:validation-api:2.0.1.Final")
     testImplementation("org.springframework:spring-webmvc:5.1.9.RELEASE")
+    testImplementation("org.springframework.security:spring-security-web:5.4.0")
     testImplementation("io.micronaut:micronaut-core:3.8.4")
     testImplementation("io.micronaut:micronaut-http:3.8.4")
+    testImplementation("io.micronaut:micronaut-security-jwt:2.0.0.M2")
```

Comment:
**nitpick:** I think you actually want `io.micronaut.security:micronaut-security:3.8.3`?

Reply:
This dependency does not work for me. The one that works: `io.micronaut:micronaut-security:2.0.0.M2`

Reply:
Are you sure? The test you have pushed don't work with any of the three packages for me.

Reply:
That is strange. The package you mentioned, leads to failing tests from my side. 😅

Reply:
Are you talking about unit tests failing or about the kt files used for comparison showing erros?

Reply:
<img width="420" alt="Screenshot 2023-04-14 at 08 06 58" src="https://user-images.githubusercontent.com/6298068/231955338-9ac48f8c-4276-438e-b2ab-ed6820a58238.png">
these tests

Reply:
And what's the cause of error? :)